guides/15_Background_Filter_Video_Processor.md

```diff
@@ -116,9 +116,24 @@ let transformDevice = new DefaultVideoTransformDevice(logger, device, processors
 
 Both `isSupported` and `create` accept _specifications_ — `BackgroundFilterSpec` and `BackgroundBlurOptions` structures — as input. These `BackgroundFilterSpec` allows you to describe the model attributes, CDN paths, and other configuration options that define how the feature should operate. The `BackgroundBlurOptions` allows you to configure runtime options like observer reporting period, logging and the amount of blur strength to apply to the video.
 
-
 Most developers will not need to provide a specification: the defaults are chosen to work well and adapt to runtime constraints. 
 
+**BackgroundBlurOptions**
+
+```typescript
+interface BackgroundBlurOptions {
+  logger?: Logger;
+  reportingPeriodMillis?: number;
+  blurStrength?: number;
+  filterCPUUtilization?: number;
+}
+```
+
+* logger: A logger to which log output will be written.
+* reportingPeriodMillis: How often the video frame processor will report observed events. 
+* blurStrength: The amount of blur that will be applied to a video stream. 
+* filterCPUUtilization: The threshold CPU utilization to trigger skipping background filter updates which will reduce the amount of CPU used by background filtering. For example, If the reporting period is set to 1000 ms and 500 ms was dedicated to processing the background filter, then the CPU utilization for that reporting period is 50%. If `filterCPUUtilization` is set to 50 it will cause a `filterCPUUtilizationHigh` event to be fired from the `BackgroundBlurVideoFrameProcessorObserver`.
```

Comment:
Should builders do anything when receiving `filterCPUUtilizationHigh`?

Reply:
I will add an example implementation and what action to take if this fires. Mainly you can disable background blur if the utilization is too high.